### PR TITLE
updated github actions to remove warnings

### PR DESCRIPTION
**Pull request by jhmjam**
Github actions changed to remove warning label

### CODE REVIEW

**Review by sanjeevz3009 — APPROVED** [review]
All all good to me =)

File: .github/workflows/pr-closed.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ jobs:
         uses: actions/checkout@v3
       - run: echo "The ${{ github.repository }} repository has been cloned to the runner."
       - name: Configure AWS credentials from Dev account
-        uses: aws-actions/configure-aws-credentials@v1
+        uses: aws-actions/configure-aws-credentials@v1-node16
```
Comment (JasonBellONS) [inline .github/workflows/pr-closed.yml:22]: Out of interest why we're specifying node16 now?

Comment (jhmjam) [reply .github/workflows/pr-closed.yml:22]: Its a weird fix, at the moment set output changes (doing it the new way) does not always actually work. Until github fix this issue the recommendation was to change the package, here is where i found this suggestion :) https://github.com/aws-actions/configure-aws-credentials/issues/521

Comment (JasonBellONS) [reply .github/workflows/pr-closed.yml:22]: ah strange, all looks good though

**Review by JasonBellONS — APPROVED** [review]
All looks good

**Pull request merged**